cli/popper/popper-complete.sh

```diff
@@ -0,0 +1,8 @@
+_popper_completion() {
+    COMPREPLY=( $( env COMP_WORDS="${COMP_WORDS[*]}" \
+                   COMP_CWORD=$COMP_CWORD \
+                   _POPPER_COMPLETE=complete $1 ) )
+    return 0
+}
+
+complete -F _popper_completion -o default popper;
```

Comment:
@UjjwalAyyangar can we place this file in `cli/extras/bash-completion.sh` instead?